apk.css

```diff
@@ -1,30 +1,53 @@
-.logo {
-	vertical-align: bottom;
-	margin-right: 15px;
+body {
+	font-family: 'Open Sans', arial, sans;
```

Comment:
`Arial` with capitalized `A`. 

`sans` isn't a typeface AFAIK. Change to `sans-serif` since it's a sans-serif.